draft-ietf-tls-tls13.md

```diff
@@ -2201,9 +2204,13 @@ hash
 
 signature
 : This field indicates the signature algorithm that may be used.
-  The values indicate anonymous signatures, RSASSA-PKCS1-v1_5
-  {{RFC3447}}, DSA {{DSS}}, and ECDSA {{ECDSA}}, respectively.  The
-  "anonymous" value is meaningless in this context but used in
+  The values indicate anonymous signatures, RSASSA-PKCS1-v1_5,
+  {{RFC3447}}, DSA {{DSS}}, ECDSA {{ECDSA}}, and
+  RSASSA-PSS respectively. Because all RSA signatures used in the
+  TLS protocol itself (as opposed to those in certificates)
```

Comment:
The "in the TLS protocol itself" is kinda vague and rather imprecise. Just be explicit and say "in signed TLS handshake messages (see {{digital-signing}})".